nexus/tests/integration_tests/instances.rs

```diff
@@ -133,6 +135,18 @@ fn get_disks_url() -> String {
     format!("/v1/disks?{}", get_project_selector())
 }
 
+async fn get_instance_disks(
+    client: &ClientTestContext,
+    name: &str,
+) -> Vec<Disk> {
+    let url = format!("/v1/instances/{name}/disks?project={}", PROJECT_NAME);
```

Comment:
unimportant nit: could also inline the `PROJECT_NAME`:
```suggestion
    let url = format!("/v1/instances/{name}/disks?project={PROJECT_NAME}");
```

Reply:
d10dcdeb6